chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -127,13 +103,16 @@
     only_if { ::File.exist?("/usr/sbin/crm") }
   end
 
-  # Stop other services
-  service "drbd" do
+  # Stop the database and corosync
```

Comment:
Style/CommentIndentation: Incorrect indentation detected (column 2 instead of 3).